templates/explore/organizations.tmpl

```diff
@@ -0,0 +1,35 @@
+{{template "base/head" .}}
+<div class="explore users">
+	<div class="ui container">
+		<div class="ui grid">
+			{{template "explore/navbar" .}}
+			<div class="twelve wide column content">
+				{{template "explore/search" .}}
+
+				<div class="ui user list">
+					{{range .Users}}
+						<div class="item">
+						  <img class="ui avatar image" src="{{.RelAvatarLink}}">
+						  <div class="content">
+						  	<span class="header"><a href="{{.HomeLink}}">{{.Name}}</a> {{.FullName}}</span>
+						    <div class="description">
+									{{if .Location}}
+										<i class="octicon octicon-location"></i> {{.Location}}
```

Comment:
Thanks, changed the link icon:

<img width="1221" alt="gogs-explore-orgs-2" src="https://cloud.githubusercontent.com/assets/251281/18168287/4784b97a-7098-11e6-83de-44213de66869.png">